src/containers/ConversionForm/index.tsx

```diff
@@ -125,6 +129,11 @@ const ConversionForm = ({
     };
 
     const handleFileSelection = async (file: UploadFile) => {
+        // 200 MB limit
+        if (file.size !== undefined && file.size > 2e8) {
```

Comment:
Is it possible for the file's size to be undefined, but the file is over 200 MB?

Reply:
I don't think so?

Reply:
I'm curious what the case is where there's a file but size is undefined? why can't it just be the size check?

Reply:
size is an optional property on the type `UploadFile` so its possibly undefined